convert2rhel/pkghandler.py

```diff
@@ -49,7 +50,7 @@ def call_yum_cmd_w_downgrades(cmd, fingerprints):
     for _ in xrange(MAX_YUM_CMD_CALLS):
         output, ret_code = call_yum_cmd(cmd, "%s" % (" ".join(
             get_installed_pkgs_by_fingerprint(fingerprints))))
-        loggerinst.info("Received return code: %s\n" % str(ret_code))
+        loggerinst.info("Received return code: {}\n".format(str(ret_code)))
```

Comment:
As we support python2.6  it requires numbers in the placeholders:
`'{0} {1}\n'.format(numb, foo)` or: `'{numb} {foo}\n'.format(numb=numb, foo=foo)`

Reply:
You're right! fixed!